index.js

```diff
@@ -31,6 +31,13 @@ var argv = require('yargs')
     default: false,
     global: true
   })
+  .option('no-verify', {
+    alias: 'n',
+    describe: 'Bypass pre-commit or commit-msg git hooks during the commit phase.',
```

Comment:
@stevemao Not sure what you mean. Do you want Noah to remove the period at the end of the description?